miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/SampleRestController.java

```diff
@@ -319,7 +319,7 @@ public SampleDto updateSample(@PathVariable("id") Long id, @RequestBody SampleDt
       throws IOException {
     Collection<SampleIdentity> matches = new HashSet<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "matches". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)